orchagent/bufferorch.cpp

```diff
@@ -21,14 +19,39 @@ type_map BufferOrch::m_buffer_type_maps = {
     {APP_BUFFER_QUEUE_TABLE_NAME, new object_map()},
     {APP_BUFFER_PG_TABLE_NAME, new object_map()},
     {APP_BUFFER_PORT_INGRESS_PROFILE_LIST_NAME, new object_map()},
-    {APP_BUFFER_PORT_EGRESS_PROFILE_LIST_NAME, new object_map()}
+    {APP_BUFFER_PORT_EGRESS_PROFILE_LIST_NAME, new object_map()},
+    {APP_BUFFER_PORT_CONFIG_TO_PG_PROFILE_TABLE, new object_map()},
+    {APP_BUFFER_PORT_CABLE_LENGTH_TABLE, new object_map()}
```

Comment:
Please feel free to ignore this comments if this is already agreed upon coding style. This is a very minor thing anyways.

I think if you add a comma (,) at the end of the last line, that doesn't change the compile result. When next developer adds new lines behind, the new diff won't carry the first 2 lines like in this diff block (- and +,).